app/views/registrations/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_for(@registration ||= Registration.new, remote: true) do |f| %>
```

Comment:
In this case, I think you shouldn't have a global variable there, but only a local one passed from the main view, which surely has it's own controller. 
Something like `= render 'form', registration: @registration`